composer.json

```diff
@@ -20,7 +20,7 @@
         "symfony/yaml": "^2.3 || ^3.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^1.6.1",
+        "friendsofphp/php-cs-fixer": "^1.6.1 || ^2.0",
```

Comment:
@core23 Nope, because php-cs-fixer may be installed globally. This is needed just for testing.